app/controllers/anonymous_feedback_controller.rb

```diff
@@ -36,17 +47,54 @@ def set_requester_on(request)
 
 private
   def index_params
-    params.permit(:path, :page, :from, :to)
+    params.permit(:path, :organisation, :page, :from, :to)
   end
 
-  def fetch_anonymous_feedback_from_support_api
-    api_params = { path_prefix: index_params[:path] }
+  def present_date_filters(api_response)
+    DateFiltersPresenter.new(
+      requested_from: index_params[:from],
+      requested_to: index_params[:to],
+      actual_from: api_response.from_date,
+      actual_to: api_response.to_date,
+    )
+  end
 
-    [:from, :to, :page].each do |sym|
-      api_params[sym] = index_params[sym] if index_params[sym]
-    end
+  def first_page
+    anonymous_feedback_index_path(index_params.merge(page: 1))
+  end
+
+  def api_params
+    {
+      path_prefix: index_params[:path],
+      organisation_slug: index_params[:organisation],
+      from: index_params[:from],
+      to: index_params[:to],
+      page: index_params[:page],
+    }.select { |_, value| value.present? }
+  end
+
+  def at_least_one_required_api_params
+    [
+      :path_prefix,
+      :organisation_slug,
+    ]
+  end
+
+  def has_required_api_params?
+    at_least_one_required_api_params.any? { |required_param|
+      api_params[required_param].present?
+    }
+  end
+
+  def fetch_anonymous_feedback_from_support_api
+    AnonymousFeedbackApiResponse.new(
+      support_api.anonymous_feedback(api_params).to_hash
+    )
+  end
 
-    support_api.anonymous_feedback(api_params)
+  def organisation_title(slug)
+    org = support_api.organisations_list.detect { |o| o["slug"] == slug }
+    org.present? ? org["title"] : slug
```

Comment:
You're right, it does seem a bit of a shame to be reinventing it.

Since we've already got the Organisations list, then yes, if it's not too much work to have those two related resources at `/organisations/` and `/organisations/:id`, I think it's the best we can do for now.